src/components/views/Login.js

```diff
@@ -4,66 +4,9 @@ import {useHistory} from 'react-router-dom';
 import {Button} from 'components/ui/Button';
 import 'styles/views/LoginRegistration.scss';
 import BaseContainer from "components/ui/BaseContainer";
-import PropTypes from "prop-types";
+import {FormField} from "../ui/FormField";
+import {getGeoLocation} from "../../helpers/getGeoLocation";
```

Comment:
good choice! i also thought , but only about that after reviewing the last PR though